### DISCUSSION TITLE
Clojure and Hello

### DISCUSSION THREAD

**Discussion by kees-** [zed-industries/zed#6446] [Feature Requests]
Hi, I'm really glad to be accepted to the alpha at the least to watch zed in development. As someone who has moved to visual studio (vscodium) it does the job and then some but I have not become a huge fan. I get an intuitive interest to dig into zed and feel I could eventually put it into part- or full-time use as things progress. I hope others feel the same here.

To start hopefully a long term and open-ended conversation (so I put this in discussions if that's ok),

Clojure tooling is make-or-break for my main editor. Centrally, connecting into a local REPL server for inline form evaluation(!), plus lisp style paredit-ing for form editing and traversal[[1](https://www.emacswiki.org/emacs/ParEdit)][[2](https://calva.io/paredit/)], I see as the core killer features for the editors that are dedicated or have robust plugins for the language. I'm sure most professional clojure devs would have their own take on their preferences with more detail.
For reference there seems to be about a half dozen notable projects, emacs/[CIDER](https://cider.mx/), VS code/[Calva](https://calva.io/), [cursive](https://cursive-ide.com/) built out of intellij, maybe vim/[fireplace](https://github.com/tpope/vim-fireplace). Not sure where [lighttable](http://lighttable.com/) or eclipse/[counterclockwise](https://marketplace.eclipse.org/content/counterclockwise) are at anymore and I'd guess atom/[proto repl](https://github.com/jasongilman/proto-repl) or [chlorine](https://github.com/mauricioszabo/atom-chlorine) will be sunset if they aren't already.

I have to assume such features, mildly uncommon languages with dedicated custom tooling, will be pretty far off and likely in the realm of extensions.

As much as I'd love to say "I'll lead the charge one day" for clojure/clojurescript I am a junior hobby dev and the range of knowledge needed to contribute to an implementation at some point is goliath.

However, I also have to imagine the collaborative and unique design of an editor intended to push the code space forward could be a game changer in many areas!

Rust has certainly interested me, I'm starting to learn some basics at least as a secondary language, partly after hearing about this editor actually. Seeing any major project with a "post-electron" trajectory is enough to be excited. My main question here currently, anywhere I can go next to learn about zed a layer beyond the public docs and browsing this repo?

I don't have requests or a call to arms right now. Thanks for just hearing out some thoughts! I might post on the most active clojure forum [clojureverse](https://clojureverse.org/) to see what discussions could be started while this editor is getting its legs.

Good luck to this team,
Kees

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
